packages/webapp/src/components/Form/Input/index.jsx

```diff
@@ -165,6 +167,7 @@ const Input = ({
           {errors}
         </Error>
       ) : null}
+      {subText ? <TextWithExternalLink link={link}>{subText}</TextWithExternalLink> : null}
```

Comment:
I don't think subtext will always be `TextWithExternalLink`. Could we have this component to take and show `subText` to make it more flexible? (and pass `<TextWithExternalLink link={link}>{subText}</TextWithExternalLink>` from the parent)

Reply:
i think there's `info` for other cases, but that didn't quite fit for this particular case